frontend/model/contracts/group.js

```diff
@@ -299,6 +299,10 @@ DefineContract({
     groupShouldPropose (state, getters) {
       return getters.groupMembersCount >= 3
     },
+    groupVotingRule (state, getters) {
+      // It's okay to use "PROPOSAL_GENERIC" as an example because all the settings are the same.
+      return getters.groupSettings.proposals[PROPOSAL_GENERIC]
```

Comment:
If memory serves, think this should be `.proposals[PROPOSAL_GENERIC].rule`?

Reply:
We also need to access `ruleSettings`, besides `rule`. Maybe the getter could have a better name. Any suggestion?

Reply:
Since our code is designed to support individualized settings on each type of proposal, and since we might re-add UI support for that in the future, we need to make sure any code we write is forward-compatible with that.

So imagine we were to re-add support for individualized proposal settings. In such a world this getter wouldn't make any sense at all.

So the getter that needs to be defined is one that would make sense in both worlds (the current one, and the possible future one).

There are various things you can do, but here's one suggestion:

```js
proposalSettings (state, getters) {
  return (proposalType = PROPOSAL_GENERIC) => {
    return getters.groupSettings.proposals[proposalType]
  }
}
```

Reply:
I know you like to predict the future when it comes to code, and although I do not agree, I can respect that.

So, to do it as you want, I have one question: At `GroupRuleSettings.js`, what `proposalType` do we pass to the getter?

Reply:
You don't need to pass in any proposalType, you can call it as an empty function. In the code I wrote above, the default, when no proposalType is passed in, will be `PROPOSAL_GENERIC`.

Reply:
I suspected that, was just to double-check, thanks!